backend/src/resolvers/project.resolver.ts

```diff
@@ -3,13 +3,15 @@ import { Arg, Authorized, Ctx, Mutation, Query } from "type-graphql";
 import Project, { NewProjectInput } from "../entities/project";
 import { Context } from "../interfaces/auth";
 import { GraphQLError } from "graphql";
+import DataSource from "../db";
 
 export default class ProjectResolver {
+	private projectRepository = DataSource.getRepository(Project);
 	@Query(() => [Project])
```

Comment:
```suggestion

        @Arg([UserRole.VISITOR, UserRole.ADMIN])
	@Query(() => [Project])
```
laisser un espace entre L9 et L10